src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -925,34 +925,45 @@ private List<ParameterAst> ParameterListRule()
             // G      parameter-list   new-lines:opt   ','   script-parameter
 
             List<ParameterAst> parameters = new List<ParameterAst>();
-            Token commaToken = null;
+            Token nextToken = null;
             while (true)
             {
                 ParameterAst parameter = ParameterRule();
                 if (parameter == null)
                 {
-                    if (commaToken != null)
+                    if (nextToken is not null)
                     {
-                        // ErrorRecovery: ??
-
-                        ReportIncompleteInput(After(commaToken),
+                        var endToken = PeekToken();
```

Comment:
Do we need to define `endToken` as a separate variable here? Can we reuse `nextToken` and if not, why?

Reply:
In both cases a new variable was used so I didn't change the behavior for the error reporting, which looks at the "nextToken". So I don't think it's strictly necessary but to keep changes minimal I decided to go with it.